portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Next time consider using a separate branch for different features. Navigation on its own, for example, so the pull requests aren't as big.